src/components/ConfigurationRow.tsx

```diff
@@ -34,13 +49,14 @@ export const getConfigurationRow = ({
   disabledReason,
   help,
   inputHelp,
-  isOverridden,
-  inheritedValue,
-  inheritSource,
-  isReadOnly,
-  value,
-  overrideValue,
+  readOnlyRenderer = (value) => <>{value}</>,
 }: Props): MainTableRow => {
+  const values = formik.values as unknown as Record<string, string | undefined>;
+  const value = values[name];
+  const isOverridden = value !== undefined;
+  const overrideValue = readOnlyRenderer(value === "" ? "-" : value);
+  const metadata = getConfigRowMetadata(formik.values, name);
```

Comment:
nice! If theese values were always fixed before then no need to pass them down as props

Reply:
They needed adjustments to generalize. Before the values were different. But with some changes the generic nature became evident and indeed now the interface could be simplified and we can calculate the values here ... Further down.